Pin voyager port

kaitmore opened the pull request:
**Please provide enough information so that others can review your pull request:**

1. Explain the **details** for making this change. What existing problem does the pull request solve? If it resolves an existing issue, be sure to use a Github [keyword](https://help.github.com/en/articles/closing-issues-using-keywords) to automatically close it.

<br/><br/>
Pin the voyager port to 30000 to make training and dev setup easier

Resolves #385 

2. What **changes to custom.yaml** are required?

<br/><br/>
/na

3. Have you documented any additional setup steps or configurations options?

<br/><br/>

4. Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.

<br/><br/>
Run `make minikube` to test

danielpcox commented:
Excellent. Looks good. Thank you, and double-thanks for updating the docs as well.

kaitmore commented:
@danielpcox thanks Daniel - I forgot to clean up a line I added while testing, could you reapprove when you get a chance?

kaitmore commented:
@danielpcox I was getting an error yesterday and pinning the k8 version was one thing I tried, but that turned out to be unrelated to my issue.

I just retested everything on the latest minikube, kubectl, and helm just to be sure. Everything worked fine - except for the line I added to the Makefile to show the voyager host/port. Not sure why that was behaving differently but I moved it back to the `show-voyager.sh` file to resolve.
